### DISCUSSION TITLE
Problem mit SD card --> SD card filesystem mount failed

### DISCUSSION THREAD

**Discussion by netonnet1984** [jomjol/AI-on-the-edge-device#2395] [Q&A] [answered]
Hallo,

ich schaffe es nicht das die SD card richtig erkannt wird.

Die LDE blinkt immer 3X + 1X.

Ich hab die card nach der Anleitung unter https://willy-tech.de/sd-karte-mit-diskpart-formatieren/ formatiert und alle Dateien der auf die card kopiert.

Ich nutze eine 16 GB SanDisk ultra. Die card wird in anderen Geräten ohne Probleme erkannt

Hat jemand eine Lösung?

Vielen Dank

**Comment by caco3** [comment]
Versuchs mal mit dem standard Formatierungstool von Windows.

**Comment by netonnet1984** [comment]
Hab ich auch schon versucht, mit dem gleichen Ergebnis --> FAT32 und alles andere auf Standard

**Comment by caco3** [comment]
Dan solltest du es mal mit einer anderen SD-Karte versuchen. Nur weils bei anderen Geräten funktioniert, ist das leider noch keine Garantie, dass der ESP32 damit klar kommt. Da gibts schon einige andere Diskussionen dazu.

**Comment by netonnet1984** [comment, marked as answer]
Hab ich eben versucht. Neue eben gekaufte SanDisk high endurance 16 GB. Geht auch nicht. Ich bestell mir mal einen neuen ESP eventuell ligt es daran.

**Comment by netonnet1984** [comment]
es war der ESP. Der neue geht.
Danke
